### PR TITLE
Adds a table capturing user and score in a match. 

### PR DESCRIPTION
**Pull request by Dawsen**
TODO:
Processes the score and user to get player ranking.
Removes unused components in the page.

### CODE REVIEW

File: src/static/add_tta_match_result.html [hunk 1]
```diff
@@ -24,6 +24,29 @@ <h2>Select a Tournament:</h2>
     <h2>Select next player to add:</h2>
     <v-select id="user_selector" v-model="selected" :options="options"></v-select>
     <button class="btn-success" onclick="addUser()">Add Player</button>
+
+    <h2>Match Result:</h2>
+    <div id="match_table">
+      <table>
+        <tr>
+          <td></td>
```
[the remark this answers is not in the document]
Comment (pg30123) [reply src/static/add_tta_match_result.html]: I meant changed to `<th>`

**Comment by pg30123** [comment]
Mostly LGTM, the demo is pretty cool!

Pre offline discussed, let's try to fix 2 small things before merging:

(1) Remove unneeded codes from the page, like the "Add Player" button and dropdown.
(2) Fix the auto selecting of tournament based on URL, might need to move the url from /add_tta_match_result back to /tournament/<tournament name>/add_tta_match_result

File: src/static/js/add_tta_match_result.js [hunk 2]
```diff
@@ -54,17 +53,25 @@ function handleTournamentsResponse(responseText) {
   initializePage();
 }
 
+function getTournamentNameFromURL() {
+  // Expected URL is "http://..../tournament/<name>/add_ffa_match_result"
```
Comment (pg30123) [inline src/static/js/add_tta_match_result.js]: nit: fix comment to be tta instead of ffa XD

Comment (Dawsen) [reply src/static/js/add_tta_match_result.js]: done

**Pull request merged**